BB8_Dome_Lights.ino

```diff
@@ -68,7 +68,7 @@ void toggleyellowLED ()
       digitalWrite (yellowLED, LOW);
 
   // remember when we toggled it
-  yellowLEDtimer = millis ();  
+  yellowLEDLastUpdate = millis ();  
   }  // end of toggleyellowLED
 
```

Comment:
You could extract the internals of each of these methods into it's own method and pass in variables for the ledpin.

```
void toggleyellowLED ()
{
  toggleLED(yellowLED);

  yellowLEDLastUpdate = millis ();  
}

void toggleLED(int ledNum) {
  int value = digitalRead (ledNum) == LOW ? HIGH : LOW;
  
  digitalWrite (ledNum, value);
}
```
I'm not sure of the Type of HIGH, LOW